lib/logstash/filters/jdbc/loader_schedule.rb

```diff
@@ -0,0 +1,39 @@
+require_relative "validatable"
+
+module LogStash module Filters module Jdbc
+  class LoaderSchedule < Validatable
+    attr_reader :schedule_frequency, :loader_schedule
+
+    private
+
+    def post_initialize
+      if valid?
+        if @cronline.seconds.is_a?(Set)
```

Comment:
Maybe a comment as to what this does? (I'm guessing that it sets the frequency that the scheduler checks for work more frequently if the cron definition means that jobs run > once per minute?)